src/components/PopUp.js

```diff
@@ -1,25 +1,44 @@
 import React, { useState } from 'react'
 import { login } from '../auth'
 import { useNavigate } from 'react-router-dom'
+import { ToastContainer, toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
 
 const PopUp = ({ setShowPopUp, setShowRegister }) => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
+  // Navigation
   const navigate = useNavigate()
 
+  // Login popup
+  const popUp = () => {
+    setShowPopUp(false)
+  }
+
+  // Toastify error
+  const errorMessage = () => {
+    toast.error('Email or password is wrong, please try again', {
+      position: 'top-center',
+      autoClose: 5000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined
```

Comment:
Configure this in the toast container.
https://fkhadra.github.io/react-toastify/introduction/